library/styleguideComponents/Wrapper/index.js

```diff
@@ -62,10 +62,7 @@ const cyanDarkTheme = {
             brand: '#80deea',
             success: '#b9f6ca',
             error: '#ff5252',
-            background: {
-                primary: '#242424',
-                secondary: '#333333',
-            },
+            background: '#303030',
```

Comment:
let's use mainBackground

Reply:
I used background to not break the continuity of de brand, success, and error, for the API it is background and then when it normalizes then it is:

background: {
        main: COLOR_WHITE,
        secondary: darken(COLOR_WHITE, 0.2),
},

Reply:
from the consumer perspective, `mainBackground` makes more sense, because I could understand it's the predominant background in my app or design, using background it will prompt the question on my head ... background??    background of what?? 

the generated background object is an implementation detail, and to be honest I don't think there will be many people trying to do more complex customization ... if we decide to move forward with the more complex customization of the backgrounds we could always allow to pass background object that it will take precedence over `mainBackground`